index.html

```diff
@@ -4,17 +4,136 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
     <!-- Add a link to your custom CSS here -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&family=Quicksand:wght@300&display=swap" rel="stylesheet"> 
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.
+    0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/main.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
 
     <title>Responsive grid project</title>
 </head>
-
 <body>
+  <header class="container header">
+    <div class="row ">
+      <button class=" offset-1 col-7 offset-md-9 col-md-2 standard-btn" type="button">Book an Appointment</button>
+      <div class=" offset-2 col-2 menu-icon">
+        <img src="./img/menu-hamburger.svg" alt="Hamburger menu">
+      </div>
+        <div class=" col-12 offset-md-1 col-md-4 head-txt">
+          <p>Maria Hope, Ph.D. - Psychologist</p>
+        </div>
+      </div>
+      
+  <!-- nav bar-->
+  <nav class="row no-gutters menu-container">
+          <div class="offset-md-6 col-md-1"><a class="menu-item" href="./index.html">Home</a></div>
```

Comment:
I would suggest using using list tags for your navigation menu rather than `<div>`. It's more appropriate semantically.

E.g.
```
<nav>
  <ul>
    <li class=""><a href="#">Home</a></li>
    <li class=""><a href="#">About</a></li>
  </ul>
</nav>
```